Makefile

```diff
@@ -3,6 +3,7 @@ INFO:=top-level Makefile for IronOS - Soldering Iron Open Source Firmware Projec
 
 
 ### global adjustable variables
+### second attempt draft PR

```

Comment:
I assume this line does not help improve the code in any way. 😂
```suggestion

```

Reply:
indeed, this was just to have something different to compile in github and check that I got the same result as in the released version hehe